README.md

```diff
@@ -210,27 +120,18 @@ Example:
 
 👤 **Author1**
```

Comment:
- [ ] Please add your name here instead of `Author1` to make Readme file more professional.